### PR TITLE
Doc: update NTFS FeedInfo properties

### PR DESCRIPTION
**Pull request by prhod**

### CODE REVIEW

File: documentation/ntfs/ntfs_fr.md [hunk 1]
```diff
@@ -506,14 +506,18 @@ feed_end_date | date | Optionnel | Date de fin de validité du jeu de données
 feed_creation_date |  date |  Optionnel | Date de génération du jeu de données
 feed_creation_time | heure | Optionnel | Heure de génération du jeu de données
 
-Le tableau ci-dessous indique les paramètres libres renseignés par Kisio Digital.
+Le tableau ci-dessous indique les paramètres libres renseignés par Kisio Digital (dépend de l'outil qui génère les données).
 
 Colonne | Type | Contrainte | Commentaire
 --- | --- | --- | ---
-feed_provider | chaine | Libre | Société/Entité fournissant le jeu de données
+feed_publisher_name | chaine | Libre | Société/Entité fournissant le jeu de données
```
Comment (pbougue) [inline documentation/ntfs/ntfs_fr.md:513]: :+1: as this is already the one we read and we don't read feed_provider ;)

**Review by pbougue — APPROVED** [review]
From code, we also use `feed_publisher_url`.
Should we list it here also?

**Comment by prhod** [comment]
> From code, we also use `feed_publisher_url`.
> Should we list it here also?

I changed it to feed_license_url because we don't have a "feed_publisher_url" today. Do you think we should have both ?

File: documentation/ntfs/ntfs_fr.md [hunk 2]
```diff
@@ -506,14 +506,18 @@ feed_end_date | date | Optionnel | Date de fin de validité du jeu de données
...
 feed_license | chaine | Libre | Licence d'utilisation des données globale du référentiel
+feed_license_url | chaine | Libre | URL associée à la license d'utilisation des données
 fusio_url | chaine | Libre | URL du système ayant généré le jeu de données
 fusio_version | chaine | Libre | Version du système ayant généré le jeu de données
+tartare_platform | chaine | Libre | Tag indiquant la plateforme qui a généré les données
+tartare_coverage_id | chaine | Libre | Id du coverage ayant généré le jeu de données
+tartare_contributor_id | chaine | Libre | Id du contributeur ayant généré le jeu de données
```
Comment (TeXitoi) [inline documentation/ntfs/ntfs_fr.md]: Je comprends pas ce que c'est.

Comment (prhod) [reply documentation/ntfs/ntfs_fr.md]: j'ai ajouté une précision, j'espère que c'est mieux ^^

**Review by nonifier — APPROVED** [review]

**Pull request merged**